src/app-layout/visual-refresh-toolbar/navigation/styles.scss

```diff
@@ -20,6 +20,7 @@
   overscroll-behavior-y: contain;
   word-wrap: break-word;
   pointer-events: auto;
+  display: flex;
```

Comment:
Why is this needed?

The default is `flex-direction: row` and I do not think we want this

Reply:
There is only one child, so it's not actually aligning anything horizontally or vertically. But I can add a more explicit definition.

Reply:
Thank makes it easier to follow, thanks